Automattic/wp-calypso
# Remove "Advanced design customization" feature from Personal plan

Part of https://github.com/Automattic/dotcom-forge/issues/1244

#### Proposed Changes

We've decided to limit Global Styles on Premium plans and above (see paYJgx-2B7-p2#comment-2977), so this PR removes the "Advanced design customization" feature from the list of features included by the Personal plan in the wordpress.com/plans page.

It also removes the "CSS customization" feature from all plans, since it is no longer needed now that all plans that includes the "Advanced design customization" feature also include the ability to change the CSS.

Before | After
--- | ---
<img width="1091" alt="Screenshot 2022-12-06 at 15 40 21" src="https://user-images.githubusercontent.com/1233880/205946403-cc21028d-22d8-4d41-9f82-69c813c6c2c0.png"> | <img width="1113" alt="Screenshot 2022-12-06 at 15 57 40" src="https://user-images.githubusercontent.com/1233880/205946430-8d4c77fc-eacc-4a97-868d-8af5c8be5c59.png">


#### Testing Instructions

- Use the Calypso live link below
- Go to "Upgrades > Plans"
- Make sure the "Advanced design customization" feature is not included in the Personal plan
- Make sure the "CSS customization" feature is not included in any plan
- Make sure the "Advanced design customization" feature for the Premium plan and above mentions the ability to change the CSS
